tests/integration/packageadd-namespace/provider/schema.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "mypkg",
+  "version": "1.0.0",
+  "namespace": "example",
```

Comment:
Nit: I'd use something more specific like `my-namespace` or whatever, I spent some brain cycles trying to understand where `example` is coming from in the test.

Reply:
Changed.